### ISSUE TITLE
certificate is expired

### ISSUE THREAD

**Issue by JoernT** [webcomponents/webcomponents.org#1255]
[Origin page](https://www.webcomponents.org/)

as of 2d Sept 2019

**Comment by kreynen** [comment]
This is also a duplicate of https://github.com/webcomponents/webcomponents.org/issues/1254

**Comment by hyperpress** [comment]
Really, this just needs a 301 redirect to www only.

**Comment by kreynen** [comment]
Not exactly.  I closed https://github.com/webcomponents/webcomponents.org/issues/1256 as a duplicate, but that issue explains that the redirection does exist. The issue is that it requires the user to accept a bad cert before they are redirected.

http://webcomponents.org/ redirects to https://webcomponents.org/ - CERT ERROR
https://webcomponents.org/ redirects to https://www.webcomponents.org/ - CERT ERROR

I did find https://github.com/webcomponents/webcomponents.org/issues/1196#issuecomment-410371389

> Yeah, the cert is till 2 August 2019 now. See you folks in one year! - @kachkaev

So here we are.
